op-service/txmgr/txmgr.go

```diff
@@ -459,7 +459,7 @@ func (m *SimpleTxManager) sendTx(ctx context.Context, tx *types.Transaction) (*t
 func (m *SimpleTxManager) publishTx(ctx context.Context, tx *types.Transaction, sendState *SendState, bumpFeesImmediately bool) (*types.Transaction, bool) {
 	l := m.txLogger(tx, true)
 
```

Comment:
Will this log the full tx hash or the minimal tx hash? When I say minimal I mean `0xaaaa...aaaa` - does it matter to you the difference?

Reply:
Just wanted it to match what's logged elsewhere as "tx", which seems to be full txhash.